### DISCUSSION TITLE
是如何与mapboxgl的相机同步

### DISCUSSION THREAD

**Discussion by xiaxiangfeng** [antvis/L7#2073] [Q&A]
请问是如何与mapboxgl的相机同步，想学习一下这个解决方案

**Comment by lzxue** [comment]
看一下 L7 的maps 模块

**Reply by xiaxiangfeng** [reply]
https://codepen.io/xiaxiangfeng/pen/KKJyvoV

看了，已经知道了，非常感谢。不过这个和我正使用的方式不同，我借鉴了threebox的方式，但是目前遇到一个问题，就是在使用THREE.InstancedMesh时模型不能完整的渲染有丢失。目前定位大概是自定义投影矩阵的问题。不知道能不能在这里得到帮助，谢谢
